packages/components/basic-table/stories/basic-table.stories.js

```diff
@@ -0,0 +1,185 @@
+					renderer: 'renderCard'
+				},
+				{
+					visibility: 'desktop-only',
+					property: 'type',
+					size: 'small',
+					value: 'Type',
+					display: 'left',
+					renderer: 'renderStatus'
+				},
+				{ visibility: 'desktop-only', property: 'participants', size: 'small', value: 'Participants' },
+				{
+					visibility: 'desktop-only',
+					property: 'newActivity',
+					value: 'New Activity',
+					renderer: 'renderMobile'
+				},
+				{
+					visibility: 'desktop-only',
+					property: 'lastActivity',
+					value: 'Last Activity',
+					size: 'medium',
+					display: 'right'
+				},
+				{
+					visibility: 'mobile-only',
+					property: 'mobileLastActivity',
+					size: 'small',
+					display: 'right',
+					renderer: 'renderUnread'
+				}
+			]);
+			const data = object('data', [
+				{
+					context: {
+						name: 'Invoice #32131',
+						description: 'Tradeshift',
+						mobileDescription: 'Tradeshift | 5 participants',
+						type: 'offer'
+					},
+					type: { status: 'ok', text: 'External' },
+					participants: 100,
+					newActivity:
+						'Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose.',
+					lastActivity: 'Yesterday, 15:01',
+					mobileLastActivity: { unread: 10, lastActivity: 'Yesterday, 15:01' },
+					id: 1
+				},
+				{
+					context: {
+						name: 'Purchase Request #1231',
+						description: 'MegaImage',
+						mobileDescription: 'MegaImage',
+						type: 'document'
+					},
+					type: { status: 'error', text: 'Private' },
+					participants: 100,
+					newActivity:
+						'Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose.',
+					lastActivity: '29 Jul, 15:01',
+					mobileLastActivity: { unread: 0, lastActivity: '29 Jul, 15:01' },
+					id: 2
+				},
+				{
+					context: {
+						name: 'GoodsReceipt #231',
+						description: 'CocaCola',
+						mobileDescription: '5 participants',
+						type: 'private'
+					},
+					type: { status: 'note', text: 'Internal' },
+					participants: 100,
+					newActivity:
+						'Every little bunny has a habit that is funny. It doesnt matter where he goes he always wrinkles up his nose.',
+					lastActivity: '2 Aug, 15:01',
+					mobileLastActivity: { lastActivity: '2 Aug, 15:01' },
+					id: 3
+				}
+			]);
+			const dir = select(
+				'dir',
+				{
+					ltr: 'ltr',
+					rtl: 'rtl'
+				},
+				'ltr'
+			);
+			const selectedIds = object('selectedIds', [1]);
+			// eslint-disable-next-line no-eval
+			const renderCard = eval(
+				text(
+					'card',
+					`(item, row) => {
+   const card = document.createElement('ts-document-card');
+   card.name = item.name;
+   card.description = item.description;
+   card.mobileDescription = item.mobileDescription;
+   card.selected = row.mobileLastActivity.unread > 0;
+   card.cardType = item.cardType;
+   return card;
+  }`
+				)
+			);
+			// eslint-disable-next-line no-eval
+			const renderStatus = eval(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make sure that this dynamic injection or execution of code is safe. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AEval)

Reply:
`NOSONAR` is a bit too long here.
Sonarqube isn't picking up on this comment and for that blocks the merge nonetheless (green build, red status due to no new issues found, with a few left behind still unresolved).
Try a shorter, one-lined `NOSONAR` comment to see if that fixes it.